pkg/notes/notes.go

```diff
@@ -189,7 +189,29 @@ func ListReleaseNotes(
 			continue
 		}
 
-		if strings.TrimSpace(note.Text) == "NONE" {
+		// exclusionFilters is a list of regular expressions that match notes text that
+		// are deemed to have no content and should NOT be added to release notes.
+		exclusionFilters := []string{
+			"^([nN][oO][nN][eE]|[nN]/[aA])$", // 'none' or 'n/a' case insensitive
```

Comment:
I think we can match case insensitive via `(?i)` here, right? It would make the regex easier to read :innocent:

Reply:
Beside this, do we want to make two exclusion filters instead of having one combining two?

Reply:
Case insensitive makes total sense. Can't believe I didn't use that to begin with ☹ I'll make the change.

I'd keep it as one for now. It makes the regex processing more efficient. If we get a lot more, then maybe we can rethink.